libc/test/src/time/strftime_test.cpp

```diff
@@ -0,0 +1,113 @@
+
```

Comment:
nit: remove this extra newline. Also fix the name on the first line of the license header